Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM dorowu/ubuntu-desktop-lxde-vnc:bionic
+LABEL maintainer="Oliver Wannenwetsch"
+
+# fix apt-dependies from lxde-vnc docker image
+RUN sudo sed -i 's|http://tw.|http://|g' /etc/apt/sources.list \
+        && sudo sed -i 's|https://tw.|https://|g' /etc/apt/sources.list
+
+# install vtorc build dependencies
+RUN apt-get update \
+        && DEBIAN_FRONTEND=noninteractive apt-get upgrade -y \
+        && DEBIAN_FRONTEND=noninteractive apt-get install -y \
+            libglib2.0-dev \
+            libgl1-mesa-dev \
+            libglu1-mesa-dev \
+            freeglut3-dev \
+            libplib-dev \
+            libopenal-dev \
+            libalut-dev \
+            libxi-dev \
+            libxmu-dev \
+            libxrender-dev \
+            libxrandr-dev \
+            libpng-dev \
+        && rm -rf /var/lib/apt/lists/*
+
+# set vtorc as autorun and desktop entry
+RUN mkdir -p /root/.config/autostart \
+        && mkdir -p /root/Desktop \
+        && echo "[Desktop Entry]" >> /root/.config/autostart/vtorc.desktop \
```

Comment:
Could you make this a multiline string pls? You can remove unwanted indentation (if needed) like this:
https://stackoverflow.com/questions/23929235/multi-line-string-with-extra-space-preserved-indentation